types/recharts/index.d.ts

```diff
@@ -207,7 +208,7 @@ export interface BarProps extends EventAttributes, Partial<PresentationAttribute
     shape?: React.ReactElement<any> | ContentRenderer<RectangleProps>;
     data?: BarData[];
     // see label section at http://recharts.org/#/en-US/api/Bar
-    label?: boolean | Label | React.SFC<LabelProps> | React.ReactElement<LabelProps> | ContentRenderer<Label>;
+    label?: boolean | Label | React.SFC<LabelProps> | React.ReactElement<LabelProps> | ContentRenderer<any>;
```

Comment:
Is `ContentRenderer<Label>` too narrow-typed? What's the problem that this change addresses?

Reply:
Label is just incorrect. These are not the props that are passed to the function. I did not have time to write the correct definitions, so I've changed it to any. Which is always better than something incorrect, I believe.